boomer/boomer.go

```diff
@@ -127,6 +131,9 @@ func (b *Boomer) runWorker(n int) {
 		TLSHandshakeTimeout: time.Duration(b.Timeout) * time.Millisecond,
 		Proxy:               http.ProxyURL(b.ProxyAddr),
 	}
+	if b.H2 {
+		http2.ConfigureTransport(tr)
```

Comment:
We need to figure if it failed because it is already HTTP2 or for another reason. If it cannot switch to HTTP2, it will silently make HTTP1 requests which is contracting to the -h2 flag. Brad gave a talk a while ago that explains how to enable/disable HTTP2: https://www.youtube.com/watch?v=FARQMJndUn0. It would be awesome if we are disabling http2 if -h2 is not, and enable it otherwise the way he recommends on the video.